data_managers/data_manager_build_kraken_database/tool_dependencies.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <set_environment version="1.0">
+        <environment_variable action="set_to" name="KRAKEN_SCRIPT_PATH">$REPOSITORY_INSTALL_DIR</environment_variable>
```

Comment:
A bug having a workaround doesn't mean the bug doesn't exist :) - that said lets merge this.